back-end/convert.py

```diff
@@ -0,0 +1,74 @@
+import logging
+import os
+
+import img2pdf
+from PIL import Image
+
+log = logging.getLogger("convert")
+
+
+def change_extension(filename, extension):
+    file, _ = os.path.splitext(filename)
+    return file + extension
+
+
+def get_filename(filepath):
+    return os.path.basename(filepath)
+
+
+def to_jpg(input):
```

Comment:
Another builtin shadowing like `file` above (and this one is in both Python 2 and 3); maybe `s/input/input_` or similar. Both of those shadows are elsewhere too, fwiw. Only putting in these two comments for now tho 😀